eng/SourceBuild.props

```diff
@@ -16,6 +16,10 @@
           DependsOnTargets="PrepareInnerSourceBuildRepoRoot"
           BeforeTargets="RunInnerSourceBuildCommand">
     <Delete Files="$(InnerSourceBuildRepoRoot).globalconfig" />
+
+    <PropertyGroup>
```

Comment:
Moving this property outside of the `RemoveInnerGlobalConfig` fixed the infinite clone issue locally for me. @jkoritzinsky - I think you should be able to keep this inner build arg, it just needs to be moved.

Reply:
I just figured out the problem. I'm overriding the property instead of appending to it. I'll fix this after lunch